libs/platform/experience/src/services/experience-data/default-experience-data.service.ts

```diff
@@ -26,15 +26,23 @@ export class DefaultExperienceDataService implements ExperienceDataService {
     if (!Object.keys(this.records).length) this.initialize();
 
     const data = Object.values(this.records);
+    // Register all components by id's
+    this.processComponent(data, cb);
```

Comment:
Isn't it better to name it `registerComponent`?

Reply:
fixed